### PR TITLE
Voting members

### PR DESCRIPTION
**Pull request by giblet22**
Changes: 
Adding new voting members and cleaned up people folder, as a number of voting members were created in that folder previously.

### CODE REVIEW

**Comment by SawanShariar** [comment]
Hi, 
I appreciate your work. Need a little correction with my name. My full namr is "S M Sawan Shariar". It will be great if you check it again.
Thank you so much. 
Sawan

**Review by SawanShariar — CHANGES_REQUESTED** [review]
Thank you so much. Need a little correction with my name if possible. My full name is "S M Sawan Shariar". It will be my great pleasure if you take the necessary step on this regard.

**Review by ramyaragupathy — APPROVED** [review]
Looks great 💯

File: _people/voting-members/sawan-shariar.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: S M Sawan Shariar
+date: 2020-05-20 17:00:00 Z
+Member Type:
+  Is Voting Member: true
+---
```
Comment (SawanShariar) [inline _people/voting-members/sawan-shariar.md:6]: Country: Bangladesh

Comment (SawanShariar) [reply _people/voting-members/sawan-shariar.md:6]: Sawan

**Review by SawanShariar — COMMENTED** [review]
Thanks

**Pull request merged**